sysmonitor@orcus/files/sysmonitor@orcus/3.2/applet.js

```diff
@@ -181,6 +182,9 @@ MyApplet.prototype = {
         graph.setDrawBorder(this.cfg_draw_border);
         graph.bg_color = this.bg_color;
         graph.border_color = this.border_color;
+        tooltip_decimals = this.getGraphTooltipDecimals(graph_idx);
```

Comment:
Hello, @OrcusCZ.

A couple of notes.

*Strict violation*. All variables need to be declared. Example: `let tooltip_decimals =...`